### DISCUSSION TITLE
from where to download?

### DISCUSSION THREAD

**Discussion by carrieleo15** [opensteno/plover#1603] [General]
I just started reading the installation instructions for Plover and was wondering from where do i download the windows installation executable?

**Comment by aerickt** [comment]
You can find the downloads in the "releases" page here: https://github.com/openstenoproject/plover/releases 
You have to scroll down a bit and click on "assets" to reveal the download links. For Windows, you're gonna want to click on the "Windows: Installer" link.

**Comment by carrieleo15** [comment]
thank you.  one would think that the links would be in the instructions
where it mentions to download the program.

**Reply by aerickt** [reply]
Perhaps it's not explicit enough, but it is linked in the installation guide here: https://github.com/openstenoproject/plover/wiki/Installation-Guide#download-plover

I've just edited the page so hopefully it's more clear.

**Comment by carrieleo15** [comment]
yeah i tried that link and there's nothing there.  It just gives me text
and another page of the installation manual.  I see no link for the
download.

**Reply by aerickt** [reply]
On that page it now says:
> [Go to this page to download the latest recommended release ](https://github.com/openstenoproject/plover/releases/latest)
 > Scroll down to "assets" and see below for which file to download for your operating system

Hopefully that should clear things up.

**Comment by carrieleo15** [comment]
If i want a windows executable, should i download  (#1592
<https://github.com/openstenoproject/plover/pull/1592>)?  it looks like
that is an update but I'm at a loss as to which one to use since there
isn't a clear pointer to a windows executable.

**Comment by carrieleo15** [comment]
i didn't see this message before I replied to your one before this one
so please disregard my last message.  We're getting closer :) am i to
download both the installer and the portable zip file?

**Reply by aerickt** [reply]
The [installation guide](https://github.com/openstenoproject/plover/wiki/Installation-Guide#install-plover-on-windows) says:

> Plover is available as an installer (recommended) or as a portable ZIP, useful for running off of a USB stick or with restricted permissions.

There's usually no reason to use the portable ZIP if you're running Plover off of your own device. I would recommend just downloading the installer and running that.

**Comment by carrieleo15** [comment]
Thanks, I'll check it out.

**Comment by carrieleo15** [comment]
do you know if there are any cheat sheets online?
